addons/auth_signup/static/src/js/reset_password.js

```diff
@@ -17,6 +17,9 @@ publicWidget.registry.SignUpForm = publicWidget.Widget.extend({
      */
     _onSubmit: function () {
         var $btn = this.$('.oe_login_buttons > button[type="submit"]');
+        if ($btn.prop("disabled")) {
+            return;
+        }
```

Comment:
I believe since latest changes this became useless. At least I removed it locally and tried to submit using the enter keystore and it didn't sent.

Reply:
I do not get this explanation.

- I tried to remove this diff, configured google recaptcha and then do a reset password, I do not see that double spinner behavior you mention
- Looking at the code:
  1. You add a spinner only in `website_cr_turnstile`, not in google recaptcha, from where comes that second spinner then ?
  2. On submit in recaptcha, it prevents the default behavior, add the token, and then resubmit, without adding a spinner or disabling the button:
  
![image](https://github.com/user-attachments/assets/4f4a8a8b-e3d6-477c-923e-dd5f5e2ffe71)

Can you explain how to get that double spinner when you remove this piece of code, with Google recaptcha ?